x-pack/plugins/infra/server/lib/alerting/metric_threshold/metric_threshold_executor.ts

```diff
@@ -322,28 +322,32 @@ const formatAlertResult = <AlertResult>(
     alertResult;
   const noDataValue = i18n.translate(
     'xpack.infra.metrics.alerting.threshold.noDataFormattedValue',
-    {
-      defaultMessage: '[NO DATA]',
-    }
+    { defaultMessage: '[NO DATA]' }
   );
-  if (!metric.endsWith('.pct'))
```

Comment:
💬 I'm not sure this is the best way to derive the correct formatter from the metric. I've seen usage of `const metricFormatter = get(METRIC_FORMATTERS, metric, METRIC_FORMATTERS.count);`.
I'm not 100% sure all cases are handled by this function, and since in this executor `metric` is typed as `string` it's difficult to figure out if the behaviour will be correct. Therefore I kept the same logic with the suffix.

Reply:
This is the standard established by Metricbeat, everything with a `.pct` needs to be treated as a percent.